clients/web/src/stylesheets/body/systemConfig.styl

```diff
@@ -7,12 +7,23 @@
   height 120px
 
 .g-settings-form-container
+  bg-color = #fcfaf3
```

Comment:
Our convention for variables in the stylus sheets has been to prefix variables with `$` to make it clear that they are variables, so let's do that here as well for consistency.